### PR TITLE
mike/add_trading_information_scope

### PR DESCRIPTION
**Pull request by michaelmueller-binary**

### CODE REVIEW

File: config/v3/account_closure/send.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Account Closure (request)  Required scope(s): [admin]",
```
Comment (aminmarashi-binary) [inline config/v3/account_closure/send.json]: Can we have a dedicated section for required scopes? title isn't a suitable place.

File: config/v3/account_closure/send.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "description": "This call allows clients to close all their accounts (including virtual-money account). It is assumed that all their accounts (excluding virtual) have no balance left.",
+    "hidden": 1,
```
Comment (mohammad-binary) [inline config/v3/account_closure/send.json]: Calls having this key shouldn't end up here. Maybe need to use the script, not manually copying them.

File: code.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ function isProduction(url) {
 }
 
 function isLocal(url) {
-    return url && url.indexOf('//localhost') > 0;
+    return url && url.match(/(\/\/localhost|github\.io)/);
```
Comment (mohammad-binary) [inline code.js]: If it's github.io then the following ternary in `getBaseUrl()` should return the else section. However, by this change, it won't handle URLs like this `https://michaelmueller-binary.github.io/websockets/api/`. Right?

https://github.com/binary-com/websockets/blob/4819c31efa50887b473af218bc31b1778831b5c7/code.js#L11

Comment (michaelmueller-binary) [reply code.js]: Probably my fault for the way I set up git hub pages  my urls looks like `https://michaelmueller-binary.github.io/api/`

File: config/v3/account_closure/send.json (hunk 1 above)
Comment (michaelmueller-binary) [reply config/v3/account_closure/send.json]: That shouldn't have been committed. it was an earlier approach I was testing, i'll fix it

**Comment by mohammad-binary** [comment]
This section also needs an update:
https://github.com/binary-com/websockets/blob/a9992e60f49a339b29ccfef405331bfbd6b769f8/_pages/guide.html#L28-L33

File: config/v3/account_closure/send.json (hunk 1 above)
Comment (michaelmueller-binary) [reply config/v3/account_closure/send.json]: fixed now

File: config/v3/account_closure/send.json (hunk 2 above)
Comment (michaelmueller-binary) [reply config/v3/account_closure/send.json]: fixed

File: _pages/applications.html [hunk 4]
```diff
@@ -74,6 +74,10 @@ <h1>Applications</h1>
                         <input id="trade-scope" type="checkbox" value="trade" />
                         <label for="trade-scope">Trade: Buy and sell contracts</label>
                     </div>
+                    <div>
+                        <input id="trading_information" type="checkbox" value="trade" />
```
Comment (michaelmueller-binary) [inline _pages/applications.html]:
```suggestion
                        <input id="trading_information" type="checkbox" value="trading_information" />
```

**Pull request merged**